documentation/src/docs/recovery-procedures/table-segment-recovery.md

```diff
@@ -0,0 +1,185 @@
+-rw-r--r--  1 osboxes osboxes  986804 Jun  3 01:49  completedTransactionsBatch-0.E-2-O-331331.71afe323-ae99-4f3d-a1f0-af4db475fef9
+-rw-r--r--  1 osboxes osboxes  128767 Jun  3 04:41 'completedTransactionsBatch-0$attributes.index.E-5-O-101208573.10e6a6e8-8665-4e57-bfe4-085eee5cf46b'
+-rw-r--r--  1 osboxes osboxes  128767 Jun  3 04:41 'completedTransactionsBatch-0$attributes.index.E-5-O-101337340.a2e5cf79-914e-4a01-9c35-b390a6f61f4a'
+-rw-r--r--  1 osboxes osboxes  128767 Jun  3 04:41 'completedTransactionsBatch-0$attributes.index.E-5-O-102496243.b032e528-589b-421b-8977-178d72128dcf'
+-rw-r--r--  1 osboxes osboxes 2241876 Jun  3 04:41  completedTransactionsBatch-0.E-5-O-1318135.8c0d3e40-bea7-4fbe-96ca-ac76c80283ad
+```
+
+Lets say the affected table segment name that you find out from step 1 is "completedTransactionsBatch-0". One would copy over the chunks of this segment( note that they are main table segment chunks and they do not have "attributes.index" string in their name). So one would copy over completedTransactionsBatch-0.E-1-O-0.b29fcb2f-e71f-4971-bc43-5c0a801c35e7, 
+completedTransactionsBatch-0.E-2-O-331331.71afe323-ae99-4f3d-a1f0-af4db475fef9 and completedTransactionsBatch-0.E-5-O-1318135.8c0d3e40-bea7-4fbe-96ca-ac76c80283ad to a directory of your choice.
+
+4) Fire up the admin-cli (assuming its configured correctly to run) and enter the below command.
+
+```
+data-recovery tableSegment-recovery <directory_where_you_copied to in step 3> <table segment name> <directory where you want to copy the output chunks to>
+
+Ex:-
+data-recovery tableSegment-recovery /foo/bar completedTransactionsBatch-0 /bar/foo/bar
+```
+
+5) Copy these generated attribute chunks back to the tier-2 directory we identified in step 2.
+
+6) Edit the metadata of the segments to reflect these chunks.
+
+    a)  For example  Lets say the chunk generated by the command in the folder is :-
+ 
+                13003178 Jun  3 02:55 completedTransactionsBatch-0$attributes.index.E-1-O-0.09cb6598-da98-43ad-8d9c-44ca6d523c4b
+
+    b)  Copy this chunk to the tier-2 directory i.e /mnt/tier2/_system/_tables/
+
+
+    c)  Remove the older set of chunks. That is from the above "ls" listed files above, one would remove:-
+
+           -rw-r--r--  1 osboxes osboxes  128767 Jun  3 04:41 'completedTransactionsBatch-0$attributes.index.E-5-O-101208573.10e6a6e8-8665-4e57-bfe4-085eee5cf46b'
+	   -rw-r--r--  1 osboxes osboxes  128767 Jun  3 04:41 'completedTransactionsBatch-0$attributes.index.E-5-O-101337340.a2e5cf79-914e-4a01-9c35-b390a6f61f4a'
+	   -rw-r--r--  1 osboxes osboxes  19876  Jun  3 04:41 'completedTransactionsBatch-0$attributes.index.E-5-O-102496243.b032e528-589b-421b-8977-178d72128dcf'
+          
+
+    d) Perform a get on the attribute index segment in the CLI
+
+       table-segment get _system/containers/storage_metadata_<owning Container ID here> <segmentName>$attributes.index <SS_Pod_IP_OWNING_THE_CONTAINER>
```

Comment:
Missing formatting.

Reply:
seems to be a formatting issue in the generated diff only.